ads/distroscale.md

```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# DistroScale
+
+## Example
+
+```html
+<amp-ad width=533 height=300
+    type="distroscale"
+    layout="responsive"
+    data-pid="1"
+    data-zid="8710">
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please [contact DistroScale](http://www.distroscale.com).
+
+__Required:__
+
+- `data-pid` - Partner ID
+- `data-zid` - Zone ID
```

Comment:
`data-tid` is for debugging purposes only and not intended for use by pubs. we'd still like the convenience but wouldn't want to make it publicized knowledge.